### PR TITLE
Update pyjwt version

### PR DESCRIPTION
**Pull request by matt852**
This updates the version of pyjwt from 1.7.1 to 2.1.0, along with any required dependencies.

I manually added in "cryptography" to pyproject.toml, as it's required for this and a few other packages.

Lastly, minor version bump from 1.2.0 to 1.3.0 as the backend dependencies changed.

### CODE REVIEW

**Comment by matt852** [comment]
I manually tested ChatOps functionality within MS Teams and Slack after the update, and found no issues.

**Review by glennmatthews — APPROVED** [review]
Looks reasonable to me - if it all works, :shipit:

**Review by jvanderaa — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -21,9 +21,10 @@ python = "^3.6"
 nautobot = "*"
 nautobot-capacity-metrics = "*"
 texttable = "^1.6.2"
-pyjwt = {extras = ["crypto"], version = "^1.7.1"}
+PyJWT = "^2.1.0"
 webexteamssdk = "^1.3"
 slack-sdk = "^3.4.2"
+cryptography = "^3.4.7"
```
Comment (dgarros) [inline pyproject.toml:27]: this library is used by a lot of other project, might be better to not pin its version to specific version but rather keep it flexible (if possible)

Comment (dgarros) [reply pyproject.toml:27]: I should have pushed harder on this one, I'm blocked because of this issue right now :(

**Pull request merged**